src/QMCWaveFunctions/LCAO/SoaAtomicBasisSet.h

```diff
@@ -33,6 +34,10 @@ struct SoaAtomicBasisSet
   using RealType        = typename ROT::RealType;
   using GridType        = typename ROT::GridType;
   using ValueType       = typename QMCTraits::ValueType;
+  using OffloadArray4   = Array<ValueType, 4, OffloadPinnedAllocator<ValueType>>;
+  using OffloadArray3   = Array<ValueType, 3, OffloadPinnedAllocator<ValueType>>;
+  using OffloadArray2   = Array<ValueType, 2, OffloadPinnedAllocator<ValueType>>;
```

Comment:
Array2 -> 2DArray?

Reply:
yes, I can change it to
`using OffloadMatrix = Matrix<ValueType, OffloadPinnedAllocator<ValueType>>;`

Reply:
I mean in general
OffloadArray2->OffloadArray2D
OffloadArray3->OffloadArray3D
OffloadArray4->OffloadArray4D

Reply:
oh, I can do that.  Should I leave 2D as Matrix or 2D array?

Reply:
Just pick one. I don't have preference.

Reply:
ok, the names are updated now